### PR TITLE
docs: updates for GA

### PR DESCRIPTION
**Pull request by blgm**
Here are a number of small edits relating to docs testing that we have done. Each commit is for one issue.

### CODE REVIEW

File: reference/gcp-postgresql.html.md.erb [hunk 1]
```diff
@@ -145,7 +156,7 @@ provisioning a csb-google-postgres service:
     <td>
       The ID of the Google Compute Engine network to which the instance is connected.
     </td>
-    <td><code>""</code> </td>
+    <td>The value the operator entered for <strong>authorized_network_id</strong> in <%= vars.ops_manager %>.</td>
```
Comment (FelisiaM) [inline reference/gcp-postgresql.html.md.erb:159]: I think the dafault for this is the id of the `authorized_network` set above. From my reading of [the code](https://github.com/cloudfoundry/csb-brokerpak-gcp/blob/343aafd8d3a6101d22b7c47016ab2422a6d10286/terraform/cloudsql/postgresql/provision/data.tf#L6)

**Comment by RichardJJG** [comment]
Let me know when this is ready for merging, please, @blgm -- it seems work is on-going.

**Comment by blgm** [comment]
Thanks @RichardJJG, I've added a clarification that I think resolves the discussion

**Comment by RichardJJG** [comment]
Thanks, all!

**Pull request merged**